plugins/org.eclipse.n4js.ui/src/org/eclipse/n4js/ui/labeling/N4JSHoverProvider.xtend

```diff
@@ -107,6 +147,15 @@ class N4JSHoverProvider extends DefaultEObjectHoverProvider {
 			return "Error generating documentation:  " + ex;
 		}
 	}
+	
+	def private String createDocString(String name, String jsdocString) {
```

Comment:
Maybe rename that to "addMDNLink" since this is what the method does, doesn't it?